src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -176,6 +176,15 @@
       "You cannot remember any memories or clarities."
     ]
   },
+  "stsjorbsmod:SpiritShieldPower": {
+    "NAME": "Spirit Shield",
+    "DESCRIPTIONS": [
+      "Reduce all damage by ",
```

Comment:
Prefer using String.format placeholders rather than concatenation for power localizations (I went through yesterday and updated all the existing cases)